core/src/test/java/org/apache/accumulo/core/metadata/schema/TabletMetadataTest.java

```diff
@@ -145,7 +145,7 @@ public void testFuture() {
         EnumSet.allOf(ColumnType.class), false);
 
     assertEquals(extent, tm.getExtent());
-    assertEquals(HostAndPort.fromParts("server1", 8555), tm.getLocation().getHostAndPort());
+    assertEquals(HostAndPort.fromParts("server1", 8555), tm.getLocation().getLocation());
```

Comment:
`getLocation().getLocation()` seems funny here. Too many types known as 'location'.

Reply:
Yes.  This is holdover from the temp interface in Ample.  I thought I had dropped this but I got side tracked.  I will drop this interface in the next commit.